<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1376</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FracFWD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QTreeView" name="treeView"/>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1376</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_NewProject"/>
    <addaction name="action_OpenProject"/>
    <addaction name="separator"/>
    <addaction name="action_Export"/>
    <addaction name="action_CropModels"/>
    <addaction name="separator"/>
    <addaction name="action_ViewModels"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Mesh">
    <property name="title">
     <string>Mesh</string>
    </property>
    <addaction name="action_NewMesh"/>
    <addaction name="action_ImportMesh"/>
    <addaction name="action_ViewMesh"/>
    <addaction name="separator"/>
    <addaction name="action_SaveMesh"/>
   </widget>
   <widget class="QMenu" name="menu_Background">
    <property name="title">
     <string>Background</string>
    </property>
    <widget class="QMenu" name="menu_WellPath">
     <property name="title">
      <string>Well Path</string>
     </property>
     <addaction name="action_ImportWellPath"/>
    </widget>
    <addaction name="action_NewBkg"/>
    <addaction name="separator"/>
    <addaction name="menu_WellPath"/>
    <addaction name="separator"/>
    <addaction name="action_AddSlab"/>
    <addaction name="action_AddEllipsoid"/>
    <addaction name="separator"/>
    <addaction name="action_ViewBkg"/>
   </widget>
   <widget class="QMenu" name="menu_SurveyDesign">
    <property name="title">
     <string>Survey design</string>
    </property>
    <widget class="QMenu" name="menu_ImportConfig">
     <property name="title">
      <string>Import Survey Config</string>
     </property>
     <addaction name="action_ImportSrc"/>
     <addaction name="action_ImportRcv"/>
    </widget>
    <widget class="QMenu" name="menuView">
     <property name="title">
      <string>View</string>
     </property>
     <addaction name="action_ViewSurvey"/>
     <addaction name="action_View_added_well_paths"/>
    </widget>
    <addaction name="action_NewSurvey"/>
    <addaction name="menu_ImportConfig"/>
    <addaction name="separator"/>
    <addaction name="menuView"/>
   </widget>
   <widget class="QMenu" name="menu_ModelGen">
    <property name="title">
     <string>Model Generator</string>
    </property>
    <addaction name="action_SlabGen"/>
    <addaction name="action_EllipsoidGen"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Guide"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Mesh"/>
   <addaction name="menu_Background"/>
   <addaction name="menu_SurveyDesign"/>
   <addaction name="menu_ModelGen"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_NewProject"/>
   <addaction name="action_OpenProject"/>
   <addaction name="action_Export"/>
  </widget>
  <action name="action_NewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_OpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_CropModels">
   <property name="text">
    <string>Crop Models</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_NewMesh">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_ImportMesh">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="action_ViewMesh">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="action_SaveMesh">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_NewBkg">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_AddSlab">
   <property name="text">
    <string>Add Slab</string>
   </property>
  </action>
  <action name="action_AddEllipsoid">
   <property name="text">
    <string>Add Ellipsoid</string>
   </property>
  </action>
  <action name="action_ViewBkg">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="action_NewSurvey">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_ImportSrc">
   <property name="text">
    <string>Source config</string>
   </property>
  </action>
  <action name="action_ImportRcv">
   <property name="text">
    <string>Receiver config</string>
   </property>
  </action>
  <action name="action_SlabGen">
   <property name="text">
    <string>Slabs</string>
   </property>
  </action>
  <action name="action_EllipsoidGen">
   <property name="text">
    <string>Ellipsoids</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Guide">
   <property name="text">
    <string>Guide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_DesignWellPath">
   <property name="text">
    <string>Design</string>
   </property>
  </action>
  <action name="action_ImportWellPath">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="action_ViewSurvey">
   <property name="text">
    <string>Survey</string>
   </property>
  </action>
  <action name="action_View_added_well_paths">
   <property name="text">
    <string>Add well paths</string>
   </property>
  </action>
  <action name="action_ViewModels">
   <property name="text">
    <string>View Models</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
